src/components/organisms/GlobalNav/GlobalNav.js

```diff
@@ -22,6 +22,10 @@ const Container = styled.nav`
     overflow-y: ${({active}) => (active ? "scroll" : "hidden")};
     height: 100%;
   }
+
+  a.button {
+    font-size: 35vw;
```

Comment:
This will undoubtedly lead to specificity confusion somewhere along the line. It is too loose a selector. I think we have already set something on buttons in the global nav link holder component. Can you investigate why is that not carrying through?